setup.py

```diff
@@ -50,7 +50,7 @@
         'numpy',
         'scipy',
         'matplotlib',
-        'tensorflow>=2.1',
+        'tensorflow-gpu==2.1',
```

Comment:
change back to tensorflow, that package is now gpu. Think this is what broke continuous integration.